Makefile

```diff
@@ -29,21 +29,35 @@ endif
 #
 # Included definitions
 #
-include ./tools/mk/Makefile.defs
-include ./tools/mk/Makefile.node_modules.defs
+
+# triton-mockcloud is not a public module, so override
+ENGBLD_DEST_OUT_PATH ?= /stor/builds
```

Comment:
Oh. I hadn't realized this is going to the private area. I think it would be fine to put it in public, but we can do that independent of this change if you like.

Reply:
I'd prefer to do that independently I think.